client/app/main/student/student.controller.js

```diff
@@ -14,6 +14,16 @@ function StudentCtrl($scope, $state, $stateParams, Student, toastr, Modal) {
     $scope.student = result.student;
     $scope.interventions = result.interventions;
     $scope.outreaches = result.outreaches;
+    Student.list({id: $stateParams.id}, function(data) {
```

Comment:
You don't need another call to the server here. the record is already sent down with the other results.
```
$scope.currentRecord = result.currentRecord;
```
or:
```
$scope.entry = _.first(result.currentRecord.entries);
```